src/codegen/templates/_grpc_interpreter.py.mako

```diff
@@ -145,7 +147,11 @@ class GrpcStubInterpreter(BaseInterpreter):
                 'This operation is not supported by the NI gRPC Device Server being used. Upgrade NI gRPC Device Server.'
             )
         if error_code is None:
-            raise errors.RpcError(grpc_error, error_message) from None
+            # Convert certain gRPC validation errors to DaqError for consistency with library version
```

Comment:
Which errors are these? Perhaps grpc-device should return ni-error metadata in these cases.